fdbserver/workloads/BulkLoading.actor.cpp

```diff
@@ -170,12 +173,20 @@ struct BulkLoading : TestWorkload {
 						if (bulkLoadTaskState.getRange() != KeyRangeRef(res[i].key, res[i + 1].key)) {
 							continue; // Ignore outdated task
 						}
-						if (bulkLoadTaskState.phase != BulkLoadPhase::Complete) {
+						if (bulkLoadTaskState.phase != BulkLoadPhase::Complete &&
+						    bulkLoadTaskState.phase != BulkLoadPhase::Error) {
 							TraceEvent("BulkLoadingWorkLoadIncompleteTasks")
 							    .setMaxEventLength(-1)
 							    .setMaxFieldLength(-1)
 							    .detail("Task", bulkLoadTaskState.toString());
-							return false;
+							return std::make_pair(errorTasks, false);
```

Comment:
When would this happen?

Reply:
It happens commonly as it is a monitor to see if all tasks complete periodically